event/main_listener.php

```diff
@@ -80,6 +82,12 @@ public function configure_media_embed($event)
 		/** @var \s9e\TextFormatter\Configurator $configurator */
 		$configurator = $event['configurator'];
 
+		$custom_sites_collection = $this->customSites->get_custom_sites_collection();
```

Comment:
Let's also add a comment for what we're doing here

Reply:
Like `// Add custom user site XML definitions to the default MediaEmbed sites object`